static/sass/mq.css

```diff
@@ -2528,3 +2528,11 @@ html[xmlns] .slides { display: block; }
 
 @viewport {
   width: device-width; }
+
+
+.element-4 ul,
+.element-2 ul{
+  height:288px;
```

Comment:
We don't use px as unit. Please use the `px2em` helper.

Also, is there a way to fix this without set `<ul>`'s height to a fixed value?